codecs-parent/codecs-btp/src/test/java/org/interledger/codecs/btp/AsnBtpErrorDataSerializationTest.java

```diff
@@ -65,20 +66,20 @@
   /**
    * Construct an instance of this parameterized test with the supplied inputs.
    *
-   * @param btpErrorBase64Bytes
-   * @param expectedRequestId
-   * @param expectedErrorCode
-   * @param expectedTriggeredAt
-   * @param expectedErrorDataBase64Bytes
+   * @param btpErrorBase64Bytes String of base64 encoded {@link BtpError} bytes
+   * @param expectedRequestId {@code long} expected request id.
+   * @param expectedErrorCode {@link BtpErrorCode} expected error code.
+   * @param expectedTriggeredAt {@code Instant} expected triggeredAt time.
+   * @param expectedErrorDataBase64Bytes String of expected base64 error bytes.
```

Comment:
minor nit (not enough to block the PR):

```suggestion
   * @param expectedErrorDataBase64Bytes A {@link String} of expected base64 error bytes.
```